### ISSUE TITLE
Nav block moving between auto menu and newly created menu lacks context

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#47019]
As reported in https://github.com/WordPress/gutenberg/issues/46939#issuecomment-1374652632 moving from auto-menu (Page List menu) to having a Nav Menu created when you insert a block is confusing without context. 

We need to provide context about the state change and why this happened.

This may be partly addressed by https://github.com/WordPress/gutenberg/issues/47010, but will require additional affordences to communicate the state change as well as the _current_ state of the "menu" currently being edit in the bloclk in terms of sync/unsynced.

Related https://github.com/WordPress/gutenberg/issues/47020

**Comment by scruffian** [comment]
I think we should probably reconsider this in the light of https://github.com/WordPress/gutenberg/issues/47058

**Comment by getdave** [comment]
But only if we can implement https://github.com/WordPress/gutenberg/issues/47058 in time. Otherwise it's still a problem...

**Comment by getdave** [comment]
I don't consider this part of the Nav block list view work. It's a wider issue we should look to solve on the Nav block.

**Comment by scruffian** [comment]
What happens here depends on what we do about https://github.com/WordPress/gutenberg/pull/48625

**Comment by scruffian** [comment]
Since we have the fallback state now, I don't think this issue is relevant anymore.
